### DISCUSSION TITLE
Where to report a bug in Github Classroom VSCode Extension?

### DISCUSSION THREAD

**Discussion by kwban** [community/community#46358] [GitHub Education]
Hi,

I would like to report a bug in Github Classroom VSCode Extension.

Where should I do so?

**Comment by cankush625** [comment]
One way is,
1. Go to the VS Code and go to Help > Report Issue.
2. From the report issue dialog, you can report a bug for VS Code extension.
3. Select the type as `Bug Report`, the source as `an extension`, and then select the extension. After that specify the relevant title and bug information.
4. Create a bug report.
